src/IssueViz.Security/Hotspots/HotspotsList/ViewModels/HotspotsControlViewModel.cs

```diff
@@ -121,11 +116,6 @@ private void SetCommands(ILocationNavigator locationNavigator)
                 var hotspot = (IHotspotViewModel)parameter;
                 locationNavigator.TryNavigate(hotspot.Hotspot);
             }, parameter => parameter is IHotspotViewModel);
-
-            RemoveCommand = new DelegateCommand(parameter =>
-            {
-                // todo fix in separate PR
-            }, parameter => parameter is IHotspotViewModel);
```

Comment:
Some of the code and tests have already been removed. This seems to be all that is left.